test/check-application

```diff
@@ -331,6 +331,23 @@ class TestApplication(testlib.MachineCase):
         else:
             b.wait_collected_text("#containers-containers .container-name", "abc")
 
+        # Test intermediate images
+        b.wait_not_present(".listing-action")
+        self.execute(auth, "echo 'FROM docker.io/library/registry:2\nRUN ls' > Dockerfile")
```

Comment:
Could this happen in `self.vm_tmpdir`, to avoid possible collisions in non-destructive tests?

Reply:
Well, kind of - We run this a root and as admin. For root it is fine, but admin cannot write into `self.vm_tmpdir`.

Reply:
The test has to `mkdir` it anyway, it could use `--mode 777`

Reply:
.. or use `mktemp -d` of course, if  you like that better.

Reply:
Ok, using `mktemp -d`